apps/gruenbuch/templates/gruenbuch_overview_page.html

```diff
@@ -19,8 +19,34 @@ <h1>{{ page.page_title }}</h1>
     {{ self.page_intro|richtext }}
 </section>
 <div id="layout-grid__area--maincontent">
-  {% for block in self.body %}
-      {{ block }}
-  {% endfor %}
+    {% for block in self.body %}
+        {{ block }}
+    {% endfor %}
+    {% if settings.apps_settings.Downloads.gruenbuch %}
+        {% with gruenbuch=settings.apps_settings.Downloads.gruenbuch %}
+        <h2>{% trans 'Downloads' %}</h2>
+        <div class="download__section">
+            <div class="download__section--start">
+                <h3>{{ gruenbuch.title }}</h3>
+                <p class="download-info__meta">
+                    {% trans 'Document' %} ({{ gruenbuch.file_size|filesizeformat }}) -
+                    {% trans 'Date:' %} {{ gruenbuch.created_at|date:"m/Y" }}
+                    <br>
+                    {% trans 'Document' %}: {{ gruenbuch.file|ext }}
```

Comment:
I think, you can use `gruenbuch.file_extension` (or `download.file_extension`) here and don't need the template tag. (But good use of a template tag!)

Reply:
ah cool, yes that is much better. Didn't know that exists already :)